build/scripts/build.ps1

```diff
@@ -273,6 +273,8 @@ function Deploy-VsixViaTool() {
         "Vsix\VisualStudioDiagnosticsWindow\Roslyn.VisualStudio.DiagnosticsWindow.vsix",
         "Vsix\VisualStudioIntegrationTestSetup\Microsoft.VisualStudio.IntegrationTest.Setup.vsix")
 
+    Get-Process -Name "devenv"
```

Comment:
No, purposefully printing for local runs or if somehow a process gets launched after we kill them